### PR TITLE
Problem: latest v1.11.x change is not included

### PR DESCRIPTION
**Pull request by mmsqe**
* rm msg, EIP150Hash, debug
* for more info: https://github.com/ethereum/go-ethereum/pull/25977/files https://github.com/ethereum/go-ethereum/pull/27087/files https://github.com/ethereum/go-ethereum/pull/27048/files

Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - (ante) [#353](https://github.com/crypto-org-chain/ethermint/pull/353) Remove blocked address decorator and support custom decorators instead.
 - (statedb) [#359](https://github.com/crypto-org-chain/ethermint/pull/359) Add `CacheContext` method to StateDB, to support efficient read-only native actions.
 - (rpc) [#375](https://github.com/crypto-org-chain/ethermint/pull/375) Refactor websocket/subscription system to improve performance and stability.
+- (deps) [#](https://github.com/crypto-org-chain/ethermint/pull/) Upgrade Go-Ethereum version to [`v1.11.6`](https://github.com/ethereum/go-ethereum/releases/tag/v1.11.6).
```
Comment (mmsqe) [inline CHANGELOG.md]:
```suggestion
- (deps) [#381](https://github.com/crypto-org-chain/ethermint/pull/381) Upgrade Go-Ethereum version to [`v1.11.6`](https://github.com/ethereum/go-ethereum/releases/tag/v1.11.6).
```

File: x/evm/keeper/config.go [hunk 2]
```diff
@@ -65,13 +65,11 @@ func (k Keeper) VMConfig(ctx sdk.Context, _ core.Message, cfg *statedb.EVMConfig
 		noBaseFee = k.feeMarketKeeper.GetParams(ctx).NoBaseFee
 	}
 
-	var debug bool
-	if _, ok := tracer.(types.NoOpTracer); !ok {
-		debug = true
+	if _, ok := tracer.(types.NoOpTracer); ok {
+		tracer = nil
```
Comment (mmsqe) [inline x/evm/keeper/config.go:69]: !ok => debug = true
ok => debug = false
tracer = nil means debug is false

**Review by yihuang — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -78,6 +78,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - (ante) [#353](https://github.com/crypto-org-chain/ethermint/pull/353) Remove blocked address decorator and support custom decorators instead.
 - (statedb) [#359](https://github.com/crypto-org-chain/ethermint/pull/359) Add `CacheContext` method to StateDB, to support efficient read-only native actions.
 - (rpc) [#375](https://github.com/crypto-org-chain/ethermint/pull/375) Refactor websocket/subscription system to improve performance and stability.
+- (deps) [#381](https://github.com/crypto-org-chain/ethermint/pull/381) Upgrade Go-Ethereum version to [`v1.11.6`](https://github.com/ethereum/go-ethereum/releases/tag/v1.11.6).
```
Comment (mmsqe) [inline CHANGELOG.md:82]: @yihuang need bump to [1.11.6](https://github.com/crypto-org-chain/go-ethereum/pull/14/files) first

**Review by yihuang — APPROVED** [review]
LGTM, there are a few golang lint issues

**Pull request merged**